WordPress/MediaEditor/MediaEditor.swift

```diff
@@ -1,103 +1,67 @@
 import UIKit
-import TOCropViewController
 
-public class MediaEditor: NSObject {
+public class MediaEditor: UINavigationController {
```

Comment:
Interesting! MediaEditor is now a UINavigationController! 
How come? Can you maybe add some class description ?

Reply:
Done in 4b4bdd47d7daf985487e01f3ae11163adc99bfe6